src/components/dropdown-menu/MenuItems.tsx

```diff
@@ -126,7 +126,11 @@ function Item<T>(props: ItemProps<T>) {
     <Menu.Item disabled={option.disabled}>
       {({ active }) => (
         <ItemDiv
-          onClick={() => onSelect(option)}
+          onClick={() => {
+            if (!option.disabled) {
+              onSelect(option);
+            }
+          }}
```

Comment:
should we remove this now that you call it above?

Reply:
I mean remove everything. You already call `onSelect` above.

Reply:
@Sebastien-Ahkrin

Reply:
i'm never calling onSelect if i remove theses line